packages/nimble-components/src/spinner/styles.ts

```diff
@@ -12,8 +12,8 @@ export const styles = css`
     ${display('inline-flex')}
 
     :host {
-        width: 16px;
         height: 16px;
```

Comment:
I have a mild preference to create a "small" design token and use it to set the height here, assuming that doesn't break any other behaviors.

This preference is just based on a desire for completeness and reducing the hardcoded magic constants in our CSS.